### DISCUSSION TITLE
Server Error when adding a Data Source

### DISCUSSION THREAD

**Discussion by aascedu** [amyasnikov/validity#109] [Q&A] [answered]
Hello !

I am not sure of where I should ask about this, maybe you have some infos for me :

![image](https://github.com/user-attachments/assets/b04b57d8-ad3b-4c28-a724-14b30a5a3e2b)

I get this error when trying to **add a data source** to **Netbox** to use Validity.
I am not sure how to give all the details etc, so if I didn't say something that is relevant to fixing this problem, please let me know.
Just in case in the file configuration/plugins.py I added what the Validity says to add :

```
PLUGINS = [
    "validity",
]

CUSTOM_VALIDATORS = {"core.datasource": ["validity.custom_validators.DataSourceValidator"]}
```

I am using Netbox v4.1.0, here is a link I found : https://github.com/netbox-community/netbox/issues/14279 that might address the problem.

Thank you so much, Arthur

**Comment by amyasnikov** [comment, marked as answer]
Handled within #110
